documentation/docs/specs.md

```diff
@@ -1,60 +1,22 @@
+---
+description: IOTA Client Library user-friendly API specifications.
+image: /img/logo/iota_mark_light.png
+keywords:
+- definition
+- message
+- return
+- types
+- parameters
+- address
+- account
+---
 # High Level API Specification
 
-## Table of Contents
```

Comment:
Is it really good to remove the overview here?

Reply:
Hi @Thoralf-M , in my opinion yes, cause Docusarus already generates a sidebar with all it's content. We can always keep it you prefer 
![image](https://user-images.githubusercontent.com/85233773/132808613-5b6ec98e-0671-46da-bee1-8292cd85a1f7.png)

Reply:
Hm but if one edits the file it's not there, is there an option to hide it in docusaurus? If not or if it's a lot work then I'm fine with having it removed

Reply:
I'll look into it.

Reply:
@Thoralf-M do you mean that you would like to have ToC while you edit the file?

As a workaround you could run a local preview of the site in your browser, would that work for your team?

Reply:
Not necessarily for editing, I thought more about just viewing the file in the directory, but it's not important because that shouldn't happen very often
If it's not easy to hide it, I'm fine with removing it

Reply:
Tbh I am a big fan of renderer-agnostic markdown files myself, but there are plenty of things that make it harder in our setup... ordering of the files, technical metadata at the beginning of the page and so on.

in VS Code you could navigate between headers, do you have it in your IDE?

![image](https://user-images.githubusercontent.com/42579543/133338839-01552ad3-1a9a-44fe-8b78-63af3713e078.png)

Reply:
Also I think we could wrap it into an HTML comment and then it shouldn't be displayed (but then I'm not sure if all IDEs would support clickable headers in a commented block)

Reply:
HI @Thoralf-M ,  I'm afraid I couldn't find any plugin to let us keep the TOC but not render it.  As @aleksei-korolev  mentioned, we could keep it as an HTML comment (though you'd need to uncomment each time to make the item clickable).